src/extension_commands.ml

```diff
@@ -534,6 +534,116 @@ module Copy_type_under_cursor = struct
     command Extension_consts.Commands.copy_type_under_cursor handler
 end
 
+module Search_by_type = struct
+  let extension_name = "Search By Type"
+
+  let ocaml_lsp_doesnt_support_search_by_type instance ocaml_lsp =
+    match
+      Ocaml_lsp.is_version_up_to_date
+        ocaml_lsp
+        (Extension_instance.ocaml_version_exn instance)
+    with
+    | Ok () -> ()
+    | Error (`Msg msg) ->
+      show_message
+        `Warn
+        "The installed version of `ocamllsp` does not support type search. %s"
+        msg
+
+  let _get_search_results ~query ~limit ~with_doc text_editor client =
```

Comment:
It's uncommon to have names starting with a `_` unless during debugging to prevent unused warnings. (the `_copy_type` one in this file is probably an oversight)

Reply:
Okay, will correct that. I was following the convention.